src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
+        ArrayList<Task> tasks = new ArrayList<>();
+        Scanner sc = new Scanner(System.in);
+        final String HORIZONTAL_LINE = " ____________________________________________________________";
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("  Hello! I'm Duke");
+        System.out.println("  What can I do for you?");
+        System.out.println(HORIZONTAL_LINE);
+        line = sc.nextLine();
+        while (!line.equals("bye")){
+            if(line.equals("list")){
+                System.out.println(HORIZONTAL_LINE);
+                System.out.println("  Here are the tasks in your list:");
+                for (int i = 0; i < tasks.size(); i++) {
+                    System.out.println("  " + (i+1) +":" + tasks.get(i));
+                }
+                System.out.println(HORIZONTAL_LINE);
+            }else if(line.startsWith("done")){
```

Comment:
I think maybe it would be better if you use case statements instead of keep on using (else if) for this type of tasks